.github/workflows/test.yml

```
@@ -34,3 +34,34 @@ jobs:
 
       - name: Test
         run: npm run test
+
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: 'npm'
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Link
+        run: npm link
+
+      - name: Test
+        working-directory: tests/integration
+        env:
+          SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+        run: |
+          npm ci
+          npm link testcafe-reporter-saucelabs
+          node testcafe-integration-test.js
```

Out of curiosity, why did you wrap the execution in a node script like this? The "test" script doesn't have any assertions, it's just a wrapper for a shell command. I'd think this is sufficient:

```suggestion
          npx testcafe chrome:headless tests --reporter saucelabs
```

---

Ah nothing specific, just trapped in create a `createTestCafe` runner so I created this script. I just realize I can do in this way 😄